docs/data-sources/aviatrix_all_transit_gateways.md

```diff
@@ -0,0 +1,87 @@
+---
+subcategory: "Multi-Cloud Transit"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_all_transit_gateways"
+description: |-
+Gets a list of all Aviatrix transit gateway's details.
+---
+
+
+# aviatrix_all_transit_gateways
+
+The **aviatrix_all_transit_gateways** data source provides details about all transit gateways created by the Aviatrix Controller.
+
+## Example Usage
+
+```hcl
+# Aviatrix All Transit Gateways Data Source
+data "aviatrix_all_transit_gateways" "foo" {
+  all_transit_gateway = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `all_transit_gateway` - (Required) True or False.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `transit_gateway_list` - A list include all transit gateways
```

Comment:
This should be reworded. Maybe `The list of all transit gateways`

Also, everything underneath should be indented to indicate that it is part of the block.

Reply:
have fixed